plugins/stamina/sh_plugin.lua

```diff
@@ -30,21 +30,10 @@ local function CalcStaminaChange(client)
 		return 0
 	end
 
-	local runSpeed
-
-	if (SERVER) then
-		runSpeed = ix.config.Get("runSpeed") + character:GetAttribute("stm", 0)
-
-		if (client:WaterLevel() > 1) then
-			runSpeed = runSpeed * 0.775
-		end
-	end
-
-	local walkSpeed = ix.config.Get("walkSpeed")
 	local maxAttributes = ix.config.Get("maxAttributes", 100)
 	local offset
 
-	if (client:KeyDown(IN_SPEED) and client:GetVelocity():LengthSqr() >= (walkSpeed * walkSpeed)) then
+	if (!client:GetNetVar("brth", false) and client:KeyDown(IN_SPEED)) then
```

Comment:
What stops stamina from going down when you are just standing still holding shift?